tests/test_periodic_plasma_wave.py

```diff
@@ -136,7 +136,7 @@
 zmax = 40.e-6    # Length of the box along z (meters)
 Nr = 64          # Number of gridpoints along r
 rmax = 20.e-6    # Length of the box along r (meters)
-Nm = 3           # Number of modes used
+Nm = 2           # Number of modes used
```

Comment:
Note: the above change was done so that the tests pass on GPU (until we generalize the current deposition and gathering for arbitrary number of azimuthal mode)